test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -81,7 +81,17 @@ var helpers = function helpers() {
             for (name in o) {
                 this.writeln(name, o.hasOwnProperty(name) ? "" : " (inherited)", ": ", o[name]);
             }
-        }
+        },
+
+        withPropertyDeleted: function withPropertyDeleted(object, propertyName, callback) {

```

Comment:
I saw this is a convention in this file, but why give it a name again when we have a name before column? may be this was written before Function.name implemented.